sda/internal/config/config_test.go

```diff
@@ -364,6 +365,99 @@ func (suite *ConfigTestSuite) TestGetC4GHKey() {
 	defer os.RemoveAll(keyPath)
 }
 
+func (suite *ConfigTestSuite) TestGetC4GHprivateKeys_AllOK() {
+	// Generate a crypth4gh keypair
+	_, privateKey, err := keys.GenerateKeyPair()
+	assert.NoError(suite.T(), err)
```

Comment:
This needs to be run for every key you want to use, otherwise the only difference will be the name of the c4gh private key.

That's why it is better to use a loop when you need multiple examples of something.

Reply:
Fixed in the commit b81ce66c